### ISSUE TITLE
Prerequisite Etcd3

### ISSUE THREAD

**Issue by racke** [hexfusion/Dancer2-Session-Etcd#1]
Doesn't seem to be available on CPAN.

**Comment by hexfusion** [comment]
Yes development should be local for a while.  I am transitioning everything to my new module Etcd3.  This transition will take place over the next few days.

https://github.com/sbatschelet/Etcd3

**Comment by hexfusion** [comment]
@racke also see https://github.com/sbatschelet/Etcd3/wiki for getting started details for quickstart etcd testing setup.
